### ISSUE TITLE
x/website: changes in code not highlighted in grettings-multiple-people tutorial

### ISSUE THREAD

**Issue by AaronWiora** [golang/go#46416]
### What did you expect to see?

On the site https://golang.org/doc/tutorial/greetings-multiple-people Step 2  in the code block:

```
// Request greeting messages for the names.
    messages, err := greetings.Hellos(names)
```

I think the font color should be blue, because this part changed from "Hello" to  "Hellos".

### What did you see instead?

Font color is black.

**Comment by mknyszek** [comment]
I think this is right, unless I'm misunderstanding the tutorial.

CC @dmitshur

**Comment by dmitshur** [comment]
I agree, this can be improved. There's already [CL 311329](https://golang.org/cl/311329) that was sent earlier that should address this issue. It just needs to be [reviewed](https://golang.org/doc/contribute#review).

**Comment by dmitshur** [comment]
As @ALTree pointed out in https://github.com/golang/go/issues/46642#issuecomment-856520117, [CL 277454](https://golang.org/cl/277454) is another open CL that would fix this issue.

**Comment by jamalc** [comment]
Closing as obsolete, golang.org has been merged into go.dev.

**Comment by fzipp** [comment]
@jamalc But the issue still exists on go.dev. Newbies seem to find it confusing enough that they create a new issue every couple of weeks.

**Comment by seankhliao** [comment]
Reopening since the issue is still there, just in a different location https://go.dev/doc/tutorial/greetings-multiple-people

**Comment by jamalc** [comment]
I'll take a look at the CLs.

**Comment by annagroebel** [comment]
I think the issue is resolved: 
<img width="1098" alt="Screenshot 2025-01-08 at 15 10 39" src="https://github.com/user-attachments/assets/af40be39-4fad-4661-9d2e-385b2426688e" />
@seankhliao @jamalc Can be closed
